### PR TITLE
[Core] Adding mIsTrimmed to BrepSurface

### PR DESCRIPTION
**Pull request by tteschemacher**
Small change to improve the process performance.

### CODE REVIEW

File: kratos/geometries/brep_surface.h [hunk 1]
```diff
@@ -91,6 +92,21 @@ class BrepSurface
         , mpNurbsSurface(pSurface)
         , mOuterLoopArray(BrepOuterLoopArray)
         , mInnerLoopArray(BrepInnerLoopArray)
+    {
+        mIsTrimmed = !(mOuterLoopArray.size() == 0 && mInnerLoopArray.size() == 0);
```
Comment (philbucher) [inline kratos/geometries/brep_surface.h:96]: if you can compute it I would not store it

Comment (tteschemacher) [reply kratos/geometries/brep_surface.h:96]: It is a flag to optimize the simulation. The computation of the flag is still better than nothing, however, it is valid only for very few cases.

**Comment by pooyan-dadvand** [comment]
Ok for me. :+1:

**Comment by tteschemacher** [comment]
Thx!

Could someone approve it, please?

**Review by philbucher — APPROVED** [review]

**Pull request merged**